src/unk_0202D938.c

```diff
@@ -0,0 +1,28 @@
+#include "unk_0202D938.h"
+#include "save.h"
+#include "constants/save_arrays.h"
+
+u32 sub_0202D938(void) {
+    return sizeof(UnkSavStruct20);
+}
+
+void sub_0202D93C(UnkSavStruct20* ptr) {
+    ptr->filler_00[0] = 0;
+    ptr->filler_00[1] = 0;
+    ptr->filler_00[2] = 0;
+    ptr->filler_00[3] = 0;
+    ptr->filler_00[4] = 0;
+    ptr->filler_00[5] = 0;
+    ptr->filler_00[6] = 0;
+    ptr->filler_00[7] = 0;
+    ptr->filler_00[8] = 0;
+    ptr->filler_00[9] = 0;
+    ptr->filler_00[10] = 0;
+    ptr->filler_00[11] = 0;
+    ptr->filler_00[12] = 0;
+    ptr->filler_00[13] = 0;
```

Comment:
This looks like an expanded `memset`.

```suggestion
    memset(ptr, 0, sizeof(UnkSavStruct20));
```

Reply:
oh, TIL that function exists